lib/takeaway.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'twilio'
+class Takeaway
+
+  attr_reader :menu, :orders, :message_sender, :total_order_value
+
+  def initialize(message_sender = TwilioSender.new)
+    @menu = {:FishAndChips => 5, :MeatAndTwoVeg => 7}
+    @orders = []
+    @message_sender = message_sender
+    @total_order_value = 0
+  end
+
+  def list_menu
+    p menu
+  end
+
+  def add_order_item item, quantity = 1
+    quantity.times do
+      orders << menu.select{|k,v| k == item}
```

Comment:
Unused block argument - `v`. If it's necessary, use `_` or `_v` as an argument name to indicate that it won't be used.<br>Space missing after comma.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.